connectivity/mbedtls/tools/importer/adjust-config.sh

```diff
@@ -117,6 +117,9 @@ conf unset MBEDTLS_SSL_TRUNCATED_HMAC
 
 conf unset MBEDTLS_PLATFORM_TIME_TYPE_MACRO
 
+# not needed for production code
+conf unset MBEDTLS_SELF_TEST
```

Comment:
Please update `adjust-no-entropy-config.sh`, too, as the base configuration file we use for no-entropy appears to have self-tests enabled by default (https://github.com/ARMmbed/mbedtls/blob/development/configs/config-no-entropy.h#L50).

Reply:
Done